app/src/main/res/values/strings.xml

```diff
@@ -253,6 +253,14 @@
     <string name="preference_tts_message_format_combined">Reads out user and message</string>
     <string name="preference_tts_message_format_key" translatable="false">tts_message_format_key</string>
     <string name="preference_tts_message_format_title">Message format mode</string>
+    <string name="preference_tts_message_filter_url_message">Reads out URLs</string>
+    <string name="preference_tts_message_filter_url_message_filter">Removes URLs from TTS</string>
+    <string name="preference_tts_message_filter_url_key" translatable="false">tts_message_filter_url</string>
+    <string name="preference_tts_message_filter_url_title">Filter URLs</string>
+    <string name="preference_tts_message_filter_emote_message">Reads out emotes</string>
+    <string name="preference_tts_message_filter_emote_message_filter">Removes emotes from TTS</string>
+    <string name="preference_tts_message_filter_emote_key" translatable="false">tts_message_filter_emote</string>
+    <string name="preference_tts_message_filter_emote_title">Filter emotes</string>
```

Comment:
I think in general `ignore` fits better instead of filter in the newly added strings

Reply:
Fixed in [16801e8](https://github.com/flex3r/DankChat/pull/309/commits/16801e8d12a3c54426e842693c8b70b082eeebd5)